.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libraries/Adafruit-PWM-Servo-Driver-Library"]
-	path = libraries/Adafruit-PWM-Servo-Driver-Library
+	path = libraries/libraries/Adafruit-PWM-Servo-Driver-Library/Adafruit-PWM-Servo-Driver-Library
```

Comment:
Does this not prevent the use of the IDE?

Also this seems kind of a weird path with each component duplicated—how come this is necessary?